run.py

```diff
@@ -608,13 +732,17 @@ def run_replanning_loop(settings, forecast_year):
             # use data directly from the last set of land use outputs.
             usim_post.create_next_iter_usim_data(settings, year)
 
+        # DO traffic assignment - but skip if using polaris as this is done along with activity_demand generation
         if traffic_assignment_enabled:
 
-            # 3. RUN BEAM
-            run_traffic_assignment(settings, year, client)
+            if travel_model == 'polaris':
```

Comment:
@jaauld14 same here, can you move this into the main `run_traffic_assignment()` method?

Reply:
Done